src/server/utils/createCircuitBreaker.js

```diff
@@ -22,12 +22,24 @@ import { registerCircuitBreaker } from '../metrics/circuit-breaker';
 
 const { rootModuleName } = getServerStateConfig();
 let eventLoopDelayThreshold = 250;
+let eventLoopDelayPercentile = 100;
 
 export const setEventLoopDelayThreshold = (n) => {
```

Comment:
This will always be called by root module even if not set resulting in n=undefined
```suggestion
export const setEventLoopDelayThreshold = (n=eventLoopDelayPercentile) => {
```